src/datagen/filters.ts

```diff
@@ -0,0 +1,14 @@
+import { Category, Row, Filter } from "./types";
+
+const millisecondsPerDay = 86400000;
```

Comment:
I think it'd be fine to add in the actual multiplications here. A minification transplier would pre-compute it for production code.

Reply:
I had actually googled this number directly.  Changed to computation.